#
# Cache RCE outages
# No output; cache files located at /var/spool/outagenotifier
# Be sure to leave a newline at the end of this file
#
*/5 * * * * root /usr/bin/cacheOutages.py

